src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,89 @@
+import { React, useState } from "react";
+
+export const LoginView = ({ onLoggedIn }) => {
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+    const [email, setEmail] = useState("");
+    const [birthday, setBirthday] = useState("");
+    const herokuRegister = "https://guarded-peak-19726.herokuapp.com/users";
+    const openLibraryLogin = "https://openlibrary.org/account/login.json";
```

Comment:
This is not needed anymore, use the myflix API for authentication and fetching the movies.